test/config/CRP4-session.data.xml

```diff
@@ -88,7 +88,6 @@
  <attr name="application_name" type="string" val="daq_application"/>
  <rel name="runs_on" class="VirtualHost" id="vlocalhost"/>
  <rel name="modules">
-  <ref class="HermesController" id="hermes_10_10_5"/>
```

Comment:
Why was the 6th wib endpoint removed?

Reply:
Because there are only 5 defined in dromap_CRP4.data.xml. Which is probably the file that needed fixing!!

Reply:
OK, I have put the 6th controller back in CRP4-session and added it to dromap_CRP4. I have also made a PR in oksconfgen to fix the dromap generation so that a regenerated xml will also have the 6th controller.